src/System.Text.Primitives/System/Text/Parsing/InvariantSigned.cs

```diff
@@ -2314,8 +2227,14 @@ public unsafe static bool TryParseInt32(char* text, int length, out int value, o
 
             public static bool TryParseInt32(ReadOnlySpan<char> text, out int value)
             {
+                return TryParseInt32(text, out value, out int charsConsumed);
+            }
+
+            public static bool TryParseInt32(ReadOnlySpan<char> text, out int value, out int charsConsumed)
+            {
                 if (text.Length < 1)
                 {
+                    charsConsumed = 0;
```

Comment:
Yes, of course. This code will likely change quite a bit, so I will address it then.